situp-core/build.gradle

```diff
@@ -31,4 +49,43 @@ jar {
         configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) }
     }
 
-}
\ No newline at end of file
+}
+
+task integrationTest(type: Test) {
+    description = 'Runs the integration tests.'
+    group = 'verification'
+    testClassesDirs = sourceSets.integrationTest.output.classesDirs
+    classpath = sourceSets.integrationTest.runtimeClasspath
+
+
+    doFirst {
+        startElasticsearch {
```

Comment:
QUES: just curious, I assume this local cluster is elasticsearch-oss? Is it possible for this framework to configure the ES to install plugins? For sink, ISM and security plugins are relevant although maybe not in the scope of this e2e test.